public/app/features/dashboard-scene/sharing/ShareButton/share-externally/EmailShare/ConfigEmailSharing/ConfigEmailSharing.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { selectors as e2eSelectors } from '@grafana/e2e-selectors';
+import { Button, Divider, Field, FieldSet, Icon, Stack, Tooltip } from '@grafana/ui';
+import { Input } from '@grafana/ui/src/components/Input/Input';
+import { contextSrv } from 'app/core/core';
+import { t, Trans } from 'app/core/internationalization';
+import { publicDashboardApi, useAddRecipientMutation } from 'app/features/dashboard/api/publicDashboardApi';
+import { validEmailRegex } from 'app/features/dashboard/components/ShareModal/SharePublicDashboard/SharePublicDashboardUtils';
+import { DashboardInteractions } from 'app/features/dashboard-scene/utils/interactions';
+import { AccessControlAction } from 'app/types';
+
+import { useShareDrawerContext } from '../../../../ShareDrawer/ShareDrawerContext';
+import ShareConfiguration from '../../ShareConfiguration';
+
+import { EmailListConfiguration } from './EmailListConfiguration';
+
+const selectors = e2eSelectors.pages.ShareDashboardModal.PublicDashboard.EmailSharingConfiguration;
+
+type EmailSharingForm = { email: string };
+
+export const ConfigEmailSharing = () => {
+  const { dashboard } = useShareDrawerContext();
+
+  const { data: publicDashboard, isError } = publicDashboardApi.endpoints?.getPublicDashboard.useQueryState(
+    dashboard.state.uid!
+  );
+  const [addEmail, { isLoading: isAddEmailLoading }] = useAddRecipientMutation();
+
+  const hasWritePermissions = contextSrv.hasPermission(AccessControlAction.DashboardsPublicWrite);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isValid },
+    reset,
+  } = useForm<EmailSharingForm>({
+    defaultValues: {
+      email: '',
+    },
+    mode: 'onSubmit',
+  });
+
+  const onSubmit = async (data: EmailSharingForm) => {
+    DashboardInteractions.publicDashboardEmailInviteClicked();
+    await addEmail({ recipient: data.email, uid: publicDashboard!.uid, dashboardUid: dashboard.state.uid! }).unwrap();
+    reset({ email: '' });
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FieldSet disabled={!hasWritePermissions || !publicDashboard?.isEnabled || isError}>
```

Comment:
When the public dashboard is paused, everything is disabled except the public/email sharing selection. Is that expected? The previous implementation allowed users to interact with the public dashboard configuration when disabled. 
With this new approach, I think it's unclear why the entire configuration is disabled on a paused public dashboard.

Reply:
This was changed, now a paused dashboard configuration is fully available.